### ISSUE TITLE
Improve accessibility for `input_selectize`

### ISSUE THREAD

**Issue by Rebekah-Chuang** [posit-dev/py-shiny#1105]
When I use VoiceOver on Mac to test the accessibility of my app, on the dropdown that uses `ui.input_select`, VoiceOver is able to read all the options in the dropdown. However, for `ui.input_selectize`, because of its more flexible feature that allows users to type and search within the options, the dropdown is now considered a text field and cannot read the options aloud for blind users. I prefer `ui.input_selectize` over `ui.input_select` because of its flexibility and design, hope the accessibility can be improved! Thanks~
